src/main/scala/de/upb/cs/uc4/hyperledger/utilities/EnrollmentManager.scala

```diff
@@ -1,42 +1,39 @@
-package de.upb.cs.uc4.hyperledger.utilities
-
-import java.nio.file.Path
-import java.util.Properties
-
-import org.hyperledger.fabric.gateway.Identities
-import org.hyperledger.fabric.sdk.security.CryptoSuiteFactory
-import org.hyperledger.fabric_ca.sdk.{EnrollmentRequest, HFCAClient}
-
-object EnrollmentManager{
-
-  System.setProperty("org.hyperledger.fabric.sdk.service_discovery.as_localhost", "true")
-
-  def enroll(ca_url : String, tlsCert : Path, walletPath : Path, username : String, password : String, organisationId : String): Unit = {
-    // wallet is target for admin certificate
-    val wallet = WalletManager.getWallet(walletPath)
-
-    // check if user already exists in my wallet
-    if (wallet.list.contains(username)) {
-      System.out.println(s"An identity for the admin user $username already exists in the wallet")
-      return
-    }
-
-    // Create a CA client for interacting with the CA
-    val props = new Properties
-    props.put("pemFile", tlsCert.toAbsolutePath.toString)
-    props.put("allowAllHostNames", "true")
-    val caClient = HFCAClient.createNewInstance(ca_url, props)
-    val cryptoSuite = CryptoSuiteFactory.getDefault.getCryptoSuite
-    caClient.setCryptoSuite(cryptoSuite)
-
-    // enroll my user
-    val enrollmentRequestTLS = new EnrollmentRequest
-    enrollmentRequestTLS.addHost("localhost")
-    enrollmentRequestTLS.setProfile("tls")
-    val enrollment = caClient.enroll(username, password, enrollmentRequestTLS)
-    val identity = Identities.newX509Identity(organisationId, enrollment)
-    wallet.put(username, identity)
-    System.out.println(s"Successfully enrolled user ${username} and imported it into the wallet")
-  }
-
+package de.upb.cs.uc4.hyperledger.utilities
+
+import java.nio.file.Path
+import java.util.Properties
+
+import org.hyperledger.fabric.gateway.Identities
+import org.hyperledger.fabric.sdk.security.CryptoSuiteFactory
+import org.hyperledger.fabric_ca.sdk.{EnrollmentRequest, HFCAClient}
+
+object EnrollmentManager{
+  def enroll(ca_url : String, tlsCert : Path, walletPath : Path, username : String, password : String, organisationId : String): Unit = {
+    // wallet is target for admin certificate
+    val wallet = WalletManager.getWallet(walletPath)
+
+    // check if user already exists in my wallet
+    if (wallet.list.contains(username)) {
+      System.out.println(s"An identity for the admin user $username already exists in the wallet")
+      return

```

Comment:
Don't use return in Scala
Use a clear flow

Reply:
```
if(a){
 handle(a)
 return
}

if(b){
 handle(b)
 return
}

if(c){
handle(c)
return
}

normalFLow()
``` 


find ich schöner als 

```
if(!a){
  if(!b){
   if(!c){
    normalFlow()
   }
   else{
  handle(c)
  }
 else{
  handle(b)
 }
else 
 handle(a)
}
```

Reply:
If we follow our Code Style, we dont use return.
And in your example case, you maybe could use a match with if statements.

Reply:
a,b,c can be independent - cases are not always possible:

something like:: 

```
if(a){
 handle(a)
} else {
 if(b){
  handle(b)
 } else {
  if(c){
   handle(c)
  } else {
    normalFLow()
  }
 }
}
```

?

Reply:
You can use:
```scala
if (a) {
  handle(a)
} else if (b) {
  handle(b)
} else if (c) {
  handle(c)
} else {
  normaleFlow()
}
```

Reply:
Can't you just match in the correct order or do I miss something?

Reply:
> Can't you just match in the correct order or do I miss something?

In general, yes, you are right.
But in the most cases (at least in Lagom) a, b, c are matchable and than can be checked
with a match which uses if conditions as guards

Reply:
applied in new push